### PR TITLE
Custom World exposure and Bloom scale (Better Nightmode)

### PR DESCRIPTION
**Pull request by bmbkr**
finally something that works right the first time lmao, set worldexposure to like 0.2 for nightmode, if u want to see your skins set modelAmbient to 100.
flat chams dont get affected by modelAmbient, use normal materials for your chams and they react perfectly :)

### CODE REVIEW

**Comment by newcommerdontblame** [comment]
Some quality stuff

**Comment by newcommerdontblame** [comment]
CSGO DRAINED ^'

drained
https://prnt.sc/uzepqs

**Comment by batininhani** [comment]
Cool feature!

**Comment by ny-4** [comment]
> Cool feature!

Thank ya!

File: Osiris/Config.h [hunk 1]
```diff
@@ -175,6 +175,13 @@ class Config {
             float green = 0.0f;
             float yellow = 0.0f;
         } colorCorrection;
```
Comment (airairair20) [inline Osiris/Config.h:177]: If

**Review by airairair20 — APPROVED** [review]

File: Osiris/Config.h (hunk 1 above)
Comment (airairair20) [reply Osiris/Config.h:177]: > If

Comment (ny-4) [reply Osiris/Config.h:177]: wot

**Comment by shakeit1337** [comment]
> CSGO DRAINED ^'
> 
> drained
> https://prnt.sc/uzepqs

@newcommerdontblame 
can you give me your velocity code?

**Comment by yaya222111xd** [comment]
thx uwu

**Pull request closed without merging**